datumaro/plugins/explorer.py

```diff
@@ -86,3 +87,9 @@ def type_check(self, item):
         if not isinstance(item.media, Image):
             raise MediaTypeError(f"Media type should be Image, Current type={type(item.media)}")
         return True
+
+    def inferenced_check(self, item):
+        for annotation in item.annotations:
+            if isinstance(annotation, HashKey) and not annotation.hash_key:
+                return True
+        return False
```

Comment:
Please revert this. We had no time for it.

Reply:
Revert this.